index.js

```diff
@@ -0,0 +1,97 @@
+const HOME_MENU_BUTTONS_ARRAY = [
+    {
+        id: "internet-contents",
+        className: "home-options",
+        innerText: "Internet Contents"
+    },
+    {
+        id: "digital-programme-list",
+        className: "home-options",
+        innerText: "Digital Programme List"
+    },
+    {
+        id: "digital-epg",
+        className: "home-options",
+        innerText: "Digital EPG"
+    },
+    {
+        id: "applications",
+        className: "home-options",
+        innerText: "Applications"
+    },
+    {
+        id: "recordings",
+        className: "home-options",
+        innerText: "Recordings"
+    },
+    {
+        id: "media",
+        className: "home-options",
+        innerText: "Media"
+    },
+    {
+        id: "settings",
+        className: "home-options",
+        innerText: "Settings"
+    },
+]
+
+const ELEMENT_IDS = {
+    // HOME_MENU
+    internetContents: "internet-contents",
+    digitalProgrammeList: "digital-programme-list",
+    digitalEpg: "digital-epg",
+    applications: "applications",
+    recordings: "recordings",
+    media: "media",
+    settings: "settings",
+    //
+}
+
+// ELEMENTS
+const appElement = document.getElementById("app")
+//
+
+// HELPERS
+const createElement = (tagType, attrObj, parentElement) => {
+    if (!tagType) {
+        throw new Error("Error 2");
+    }
+
+    const newElement = document.createElement(tagType)
+
+    if (attrObj) {
+        Object.entries(attrObj).forEach((entry) => {
```

Comment:
here you could descructurize entry

Reply:
and you don't need {} in the end of line

Reply:
Same

Reply:
```Object.entries(attrObj).forEach(([key, value) => {```

Reply:
Solved